lib/nexmo_developer/app/services/translator/translator_coordinator.rb

```diff
@@ -0,0 +1,18 @@
+module Translator
+  class TranslatorCoordinator
+    attr_accessor :paths
+
+    def initialize(params = {})
+      @paths = params.fetch(:paths)
+    end
+
+    def organize_jobs
+      requests = create_requests
```

Comment:
you don't need this local variable here.
You can do everything in one line, just rename `#create_requests` to `#requests`

Reply:
👍